samples/Directory.Build.props

```
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsTestGenerationSrcProject Condition="!$(MSBuildProjectName.EndsWith('.Tests'))">true</IsTestGenerationSrcProject>
+    <IsMgmtLibrary Condition="!$(MSBuildProjectName.EndsWith('.Tests')) and ($(MSBuildProjectName.StartsWith('AzureSample.ResourceManager')) or $(MSBuildProjectName.StartsWith('Mgmt')))">true</IsMgmtLibrary>
```

since this is for sample 
```suggestion
    <IsMgmtLibrary Condition="!$(MSBuildProjectName.EndsWith('.Tests')) and ($(MSBuildProjectName.StartsWith('AzureSample.ResourceManager')))">true</IsMgmtLibrary>
```